docker-compose-local-chain.yml

```diff
@@ -0,0 +1,32 @@
+version: '3'
+services:
+  cc_alice:
+    container_name: cc-alice
+    image: "centrifugeio/centrifuge-chain:${CC_DOCKER_TAG:-parachain-latest}"
+    ports:
+      - "30355:30333"
+      - "9936:9933"
+      - "9946:9944"
+    volumes:
+      - type: bind
+        source: ./res/rococo-local.json
+        target: /chainspec.json
+        read_only: true
+    command: >
+      --chain="${CC_CHAIN_ID:-development-local}" 
+      --alice
+      --parachain-id="2000"
```

Comment:
```suggestion
      --parachain-id="${CC_PARA_ID:-2000}"
```

Reply:
I opted to not do that since ports would need to be updated accordingly too

Reply:
ok 👍